src/main/java/org/fundacionjala/at13/authenticator/Auth.java

```diff
@@ -0,0 +1,75 @@
+package org.fundacionjala.at13;
+
+import java.util.Scanner;
+
+public class Auth {
+
+    public static void main(String[] args) {
+
+        int n = 0;
+        Scanner s1 = new Scanner(System.in);
+        Scanner s2 = new Scanner(System.in);
+        System.out.print("Enter no. of elements you want in array:");
+        n = s1.nextInt();
+        String usernames[] = new String[n];
+        System.out.println("Enter all the usernames:");
+        for (int i = 0; i < n; i++) {
+            usernames[i] = s2.nextLine();
+        }
+
+        System.out.println(validation(usernames));
+    }
+
+    // validation
+
+    public static boolean validation(String[] array) {
+        for (int i = 0; i < array.length; i++) {
+            if ((sizeUsername(array[i]) && itHasLowerCase(array[i]) && itHasNumber(array[i]) && itHasOnlyNumberAndLowerCaseLetter(array[i]))==false) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    // it is between 6-10 characters long
```

Comment:
On the description some ends in dot, others end with semi colon, it should be the same for all.